tuxemon/menu/input.py

```diff
@@ -201,11 +202,13 @@ def add_input_char_and_pop(self, char: str) -> None:
         self.client.pop_state()
 
     def add_input_char(self, char: str) -> None:
+        self.counter += 1
         if (
             self.char_limit is None
             or len(self.input_string) <= self.char_limit
         ):
-            self.input_string += char
+            # removes A at the end of the name
+            self.input_string += char if self.counter > 1 else ""
```

Comment:
I don't really like this implementation to be honest, I don't think we need a counter there in fact, because the only thing we care about is whether this method is called for the first time or not.
So instead, I think we need a **flag**.

Something like `self.is_first_input`, that would be initially `True`, and then assigned to `False` at middle of the first call of this method.

Reply:
true, it looks better